infrastructure/terraform/auth-service.tf

```diff
@@ -77,6 +77,13 @@ locals {
         "queue:get-artifact:*",
       ]
     },
+    {
+      clientId    = "static/taskcluster/worker-manager"
+      accessToken = "${random_string.worker_manager_access_token.result}"
+      description = "..."
+
+      scopes = []
```

Comment:
Why don't we add `'worker-manager:manage-worker-configuration:*` and `worker-manager:manage-worker-configuration:*`?

Reply:
A service has its own scopes, if that makes any sense. I think it will definitely need some more scopes to work correctly, but this at least gets it running.